DOCUMENTATION.md

```diff
@@ -1,17 +1,69 @@
 # Documentation
 
-Protogenie is a tool developed around configuration files written in XML. This design choice was made
-to allow for validation and control of configuration (this way, if your configuration file is valid but
-it does not do what it should do, we know it's a bug !).
+The purpose of _Protogenie_ is to prepare data to train a lemmatiser and/or POS tagger.
+All the training data is split into three different files:
```

Comment:
To avoid confusion "All available data [...] three different folders" (A second command is responsible for "glueing" things together ?

Reply:
corrigé. Je rajoute _A second command is responsible for "glueing" things together_ aussi?

Reply:
Peut-être entre parenthèse ? Je ne sais pas...

Reply:
Ou un deuxième paragraphe (à ce moment là, dans ton exemple je mettrais mydata1.tsv, mydata2.tsv)

Reply:
mis entre parenthèses